packages/react/src/components/Facepile/Facepile.test.tsx

```diff
@@ -181,49 +176,46 @@ describe('Facepile', () => {
       />,
     );
 
-    expectOne(wrapper, '.ms-Facepile-addButton .ms-Persona-coin.ms-Persona--size32');
-    expectOne(wrapper, '.ms-Facepile-overflowButton .ms-Persona--size32');
+    const addButton = getAllByRole('button')[0];
+    const overflowButton = getAllByRole('button')[1];
 
-    const faces = findNodes(wrapper, '.ms-Facepile-person .ms-Persona-coin');
+    expect(addButton.firstElementChild!.firstElementChild!.className).toContain('ms-Persona--size32');
```

Comment:
Is there any way to get this that's less reliant on the current DOM structure, like by text or aria-label or even classname?

Reply:
Unfortunately no text or aria-label, went with querying for a classname instead